src/App.js

```diff
@@ -6,19 +6,21 @@ import { Navbar } from "./components/navbar";
 /** import all pages into application */
 import { HomePage } from "./pages/homepage";
 import { DashboardPage } from "./pages/dashboard";
+import NotFoundPage from "./pages/NotFound";
```

Comment:
please do an object export like done for DashboardPage

Reply:
Okay